go_modules/spec/dependabot/go_modules/file_updater/go_mod_updater_spec.rb

```diff
@@ -225,13 +225,19 @@
             # OpenAPIV2 has been renamed to openapiv2 in this version
             let(:dependency_version) { "v0.5.1" }
 
-            it "raises a DependencyFileNotResolvable error" do
+            it "does not raises a DependencyFileNotResolvable error" do
               error_class = Dependabot::DependencyFileNotResolvable
               expect { updater.updated_go_sum_content }.
-                to raise_error(error_class) do |error|
+                to_not raise_error(error_class) do |error|
```

Comment:
👋 @fatih: Can you help me understand why this behavior is changing in this pull request? Since we're only changing code related to `go mod tidy`, it's not yet clear to me why this test needs to change.

Reply:
This particular change was caused previously by `go mod tidy`. Because we don't return an error anymore, this test fails as it expects an error. So I either had to remove it, or change it so it doesn't return an error.

Reply:
Thanks, @fatih. Since https://github.com/dependabot/dependabot-core/pull/2627 introduced this test, and since that PR doesn't mention anything about `go mod tidy`, I'd like to ask @jurre to take a look before we ship this PR.

@jurre: Given your context on https://github.com/dependabot/dependabot-core/pull/2627, should we be changing this test here, or would it make more sense to delete it entirely?

Reply:
I believe this case is only caught by `go mod tidy` where the import path doesn't match the exported package name, `import` here: https://github.com/dependabot/dependabot-core/blob/7ce0ebf795d757c62893e60a47a53f07e2efaee0/go_modules%2Fspec%2Ffixtures%2Fprojects%2Frenamed_package%2Fmain.go#L4 and `package` export: https://github.com/google/gnostic/blob/master/openapiv2/OpenAPIv2.go#L17

I think we can delete this test case and rely on the below `updates the go.mod` as this would fail if we raised a resolvability error.

Reply:
Yep, @feelepxyz is right 👍👍